### PR TITLE
CMR-7735 Adds EDL SIDs lookup

### PR DESCRIPTION
**Pull request by daniel-zamora**
This PR allows user groups in EDL to be used as SIDs for CMR permissions.

### CODE REVIEW

File: transmit-lib/src/cmr/transmit/urs.clj [hunk 1]
```diff
@@ -85,6 +89,27 @@
...
+  [context user-id]
+  (let [{:keys [status body]} (request-with-auth context {:url-fn #(group-search-url %)
+                                                          :method :get
+                                                          :raw? true
+                                                          :http-options {:query-params
+                                                                         {:user_ids
+                                                                          user-id}}})]
+    (info "DEBUGz1" status)
+    (info "DEBUGz2" body)
+    (info "DEBUGz3" user-id)
```
Comment (jaybarra) [inline transmit-lib/src/cmr/transmit/urs.clj:103]:
```suggestion
```

**Review by jaybarra — APPROVED** [review]
Should just drop the debugs, looks good otherwise.

File: transmit-lib/src/cmr/transmit/urs.clj [hunk 2]
```diff
@@ -85,6 +89,27 @@
...
+                                                          :method :get
+                                                          :raw? true
+                                                          :http-options {:query-params
+                                                                         {:user_ids
+                                                                          user-id}}})]
+    (info "DEBUGz1" status)
+    (info "DEBUGz2" (pr-str body))
+    (info "DEBUGz3" user-id)
+    (when-not (= 200 status)
+      (info (format "Cannot get group info for username [%s] in URS. Failed with status code [%d].
```
Comment (ygliuvt) [inline transmit-lib/src/cmr/transmit/urs.clj:105]: This should be logged as `error`.

File: transmit-lib/src/cmr/transmit/urs.clj [hunk 3]
```diff
@@ -85,6 +89,27 @@
...
+(defn get-edl-groups-by-username
+  "Returns groups associated with a username"
+  [context user-id]
+  (let [{:keys [status body]} (request-with-auth context {:url-fn #(group-search-url %)
+                                                          :method :get
+                                                          :raw? true
+                                                          :http-options {:query-params
+                                                                         {:user_ids
+                                                                          user-id}}})]
+    (info "DEBUGz1" status)
```
Comment (ygliuvt) [inline transmit-lib/src/cmr/transmit/urs.clj:101]: Remove these debug logging statements.

**Review by ygliuvt — APPROVED** [review]

**Pull request merged**